save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/VulnerabilityMetadataService.kt

```diff
@@ -153,7 +158,11 @@ class VulnerabilityMetadataService(
         ): VulnerabilityMetadata = apply {
             summary = entry.summary?.shorten(SUMMARY_LENGTH) ?: entry.details?.take(SUMMARY_LENGTH_FROM_DETAILS)?.let { "$it$ELLIPSIS" } ?: "Summary not provided"
             details = entry.details ?: "Details not provided"
-            severityNum = entry.severity?.firstOrNull()?.let { getScore(it) } ?: 0f
+            severityNum = entry.severity?.find { it.type == SeverityType.CVSS_V3 }?.let {
```

Comment:
pls move this logic to a separate method (it is common with above code)

Reply:
Done